templates/repo/release/list.tmpl

```diff
@@ -20,7 +20,9 @@
 								<span class="commit">
 									<a class="gt-mono muted" href="{{$.RepoLink}}/src/commit/{{.Sha1}}" rel="nofollow">{{svg "octicon-git-commit" 16 "gt-mr-2"}}{{ShortSha .Sha1}}</a>
 								</span>
-								{{template "repo/branch_dropdown" dict "root" $ "release" .}}
+								<div class="release-branch-button">
```

Comment:
I guess it only needs a "gt-mt-xxx" here.

And it's seems fragile to fix the "repo/branch_dropdown" 's margin-right from it's container.

Reply:
Can't do fine-grained alignment with helper classes but I will try.

Reply:
Not possible I'm afraid, all fomantic buttons comes with `margin-right` attached, which slightly misaligns them. Could be removed but then we need to manually add `gap` between all buttons etc.

Reply:
Done in 81a0c0f6d8a6e80a28a3bd47ba29179d884235bd